YonahDiaz-StockExchange/js/company.js

```diff
@@ -0,0 +1,118 @@
+let urlParams = new URLSearchParams(window.location.search);
+let stringUrlParams = urlParams.toString();
+let companySymbol = stringUrlParams.slice(7);
+
+let url =
```

Comment:
instead of rewriting the base url of the server API (https://stock-exchange-dot-full-stack-course-services.ew.r.appspot.com/), you should store it inside a global variable and use it.